src/main/kotlin/no/nav/familie/ef/søknad/integration/SøknadClient.kt

```diff
@@ -9,19 +9,22 @@ import org.springframework.web.client.RestOperations
 import org.springframework.http.HttpEntity
 import org.springframework.http.HttpHeaders
 import org.springframework.http.MediaType
-
-
+import no.nav.familie.http.client.NavHttpHeaders
+import no.nav.familie.log.mdc.MDCConstants.*
+import org.slf4j.MDC
 
 
 @Service
 internal class SøknadClient(operations: RestOperations,
-                   private val config: MottakConfig) : PingableRestClient(operations, config.pingUri) {
+                            private val config: MottakConfig) : PingableRestClient(operations, config.pingUri) {
 
-    fun sendInn(søknadDto: SøknadDto): KvitteringDto {
+    val NAV_CALLID_HEADER = NavHttpHeaders.NAV_CALLID.asString()
 
+    fun sendInn(søknadDto: SøknadDto): KvitteringDto {
         val headers = HttpHeaders()
         headers.contentType = MediaType.APPLICATION_JSON
         headers.set(config.username, config.password)
+        headers.set(NAV_CALLID_HEADER, MDC.get(MDC_CALL_ID))
```

Comment:
Når vi får inn en request opprettes en MDC (mapped diagnostic context) hvor vi putter variabler (typisk id'er), som avsluttes når requesten blir besvart. Requester som gjøres innad i denne contexten må aktivt ta i bruk variabler fra contexten. 

Snakket litt med Bent om bruk av interceptors vs filters. Sånn jeg har forstått det er interceptors mer generelle, og filters først og fremst ment for web layer (som jo er vårt case her). Ved bruk av interceptors kunne man nok unngått å måtte legge på callId her, men pga funksjonalitet i felles `LogFilter` tror jeg vi ble enige om at det var hensiktsmessig å bruke filters.